### PR TITLE
new function to convert missing epochs to a logical index

### PR DESCRIPTION
**Pull request by dominikbach**

### CODE REVIEW

**Comment by teddychao** [comment]
The newly added test function needs to be added to the test suit. I can do it here.

File: src/pspm_epochs2logical.m [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+%   Introduced in PsPM 6.2
+%   Written in 2024 by Dominik Bach (Uni Bonn)
+
+if nargin > 2 
+    epochs = pspm_time2index(epochs, sr);
+end
+
+index = zeros(datalength, 1);
+if ~isempty(epochs)
+    for k = 1:size(epochs, 1)
```
Comment (teddychao) [inline src/pspm_epochs2logical.m:27]: I have a better solution...
Simply:
`C=cellfun(@(x,y)x:y,num2cell(epochs(:,1)),num2cell(epochs(:,2)),'uni',false)` and `index([C{:}])=1`.
I dont quite understand what `index=index(1:datalength)` means...

**Review by teddychao — COMMENTED** [review]
Think my solution may be faster? Just for your information

File: src/pspm_epochs2logical.m (hunk 1 above)
Comment (dominikbach) [reply src/pspm_epochs2logical.m:27]: This might work but it is a lot more complicated to read - and speed is not an issue here.

**Review by teddychao — APPROVED** [review]

**Pull request merged**